shared/source/os_interface/windows/wddm/wddm.cpp

```diff
@@ -1111,4 +1110,24 @@ void Wddm::createPagingFenceLogger() {
         residencyLogger = std::make_unique<WddmResidencyLogger>(device, pagingFenceAddress);
     }
 }
+
+PhysicalDevicePciBusInfo Wddm::getPciBusInfo() const {
+    D3DKMT_ADAPTERADDRESS adapterAddress;
+    D3DKMT_QUERYADAPTERINFO queryAdapterInfo;
+
+    queryAdapterInfo.hAdapter = getAdapter();
+    queryAdapterInfo.Type = KMTQAITYPE_ADAPTERADDRESS;
+    queryAdapterInfo.pPrivateDriverData = &adapterAddress;
+    queryAdapterInfo.PrivateDriverDataSize = sizeof(adapterAddress);
+
+    auto gdi = getGdi();
+    UNRECOVERABLE_IF(gdi == nullptr);
+
+    if (gdi->queryAdapterInfo(&queryAdapterInfo) == STATUS_SUCCESS) {
```

Comment:
this query seems to be redundant, IMO you can reuse adapterBDF from Wddm. See here as a reference for tests: https://github.com/intel/compute-runtime/blob/master/opencl/test/unit_test/windows/os_interface_tests.cpp#L31